src/helpers/slack.ts

```diff
@@ -0,0 +1,109 @@
+const { Reaction, ErrorCode } = require('@slack/web-api');
+import SLACK_MESSAGE_BLOCKS from './files/welcomeMessageBlocks.json';
+import { SlackEvent, SlackWebClient } from './types';
+const fs = require('fs');
+
+
+function getEmojisToReactWith(text: string): Array<string> {
+    const emojisToReactWith: Array<string> = [];
+
+    let rawdata = fs.readFileSync('./helpers/files/emojis.json');
+    let emojis: Record<string, Array<string>> = JSON.parse(rawdata);
+
+    for (const [emoji, keywords] of Object.entries(emojis)) {
+        for (const keyword of keywords) {
```

Comment:
this 2 layer loop will be a performance (currently O(n) can come down to O(1) or something less resource intensive)issue when there are lots of emojis/keywords. Probably doing a data structure change to be something like this would be good :

```
const keywords = emojis[keyword];
for(cons keyword of keywords){

...
}
```

Reply:
This decreases the cyclomatic complexity too :).

Reply:
I have tried to make it simpler, please let me know if there is a better way 🙈

Reply:
For now make it work.